tests/components/application-connector/resources/charts/gateway-test/templates/k8s-test.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: Application
+metadata:
+  name: k8s-test
```

Comment:
Maybe renaming ApplicationCRD to `kubernetes-resources-error-handling` is a good idea?